pkg/kubelet/api/v1alpha1/runtime/api.proto

```diff
@@ -761,8 +763,12 @@ message Image {
     repeated string repo_digests = 3;
     // Size of the image in bytes.
     optional uint64 size = 4;
-    // User that will run the command(s).
-    optional string user = 5;
+    // UID that will run the command(s). This is used as a default if no user is
+    // specified when creating the container
```

Comment:
... And that's not documented here, only in the code. They're also mutually exclusive, which clarifies that as well.